process/block/metablock.go

```diff
@@ -478,6 +499,15 @@ func (mp *metaProcessor) CommitBlock(
 		log.LogIfError(errNotCritical)
 	}
 
+	crossCheckBlockHeight := ""
+	for i := 0; i < len(headersNonce); i++ {
+		if headersNonce[i] != uint64(0) {
```

Comment:
this will prevent showing blocks with nonce 0

Reply:
Removed this code.